p4include/v1model.p4

```diff
@@ -340,6 +348,7 @@ enum HashAlgorithm {
 }
 
 @deprecated("Please use mark_to_drop(standard_metadata) instead.")
+@packetState
```

Comment:
To be nit-picky here, the reason this is deprecated in v1model is because it isn't a legal P4_16 extern function at all.  It violates the one restriction that externs have -- it accesses P4_16 program variables that are not parameters.